scene/gui/tree.cpp

```diff
@@ -2448,7 +2458,14 @@ void Tree::_gui_input(Ref<InputEvent> p_event) {
 	}
 
 	Ref<InputEventMouseButton> b = p_event;
-
+	
+	if(is_dragExited()){
```

Comment:
Wrong indentation and style, see http://docs.godotengine.org/en/latest/community/contributing/code_style_guidelines.html